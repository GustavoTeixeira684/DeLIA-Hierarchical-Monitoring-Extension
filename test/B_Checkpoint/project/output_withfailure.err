Any local data was saved: No such file or directory
DeLIA0x5703c85ebd70
DeLIA0x5703c85edc30
DeLIA0x5703c85edc90
DeLIA0x5703c85edcc0
Any local data was saved: No such file or directory
DeLIA0x60f58c573d70
DeLIA0x60f58c575c30
DeLIA0x60f58c575c90
DeLIA0x60f58c575cc0
Any local data was saved: No such file or directory
DeLIA0x55a9eb050ab0
DeLIA0x55a9eb052970
DeLIA0x55a9eb0529d0
DeLIA0x55a9eb052a00
this->fDataGlobal.size()4
this->fDataGlobal.max_size()1152921504606846975
