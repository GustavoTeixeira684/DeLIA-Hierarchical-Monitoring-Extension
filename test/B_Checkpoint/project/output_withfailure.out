PASSOU AQUI FDP!
============================================
-----	 FAULT TOLERANCE 	-----
 FT_FOLDER 				 = ./data
 CHECKPOINTING_TIME_GLOBAL 		 = -1
 CHECKPOINTING_TIME_LOCAL 		 = -1
 CHECKPOINTING_TIME_DIFF_LOCAL 		 = -1
 CHECKPOINTING_GLOBAL_ITERATION 	 = 1
 TIME_SHARING_LOCAL_DATA_NEIGHBOR 	 = -1
 CHECKPOINTING_LOCAL_RECENT 		 = 1
 WORKLOAD_SCHEDULER 			  = DS
 WORKLOAD_SCHEDULER_LEADER 		 = -1
 TRIGGER_SIGNAL 			 = 0
 TRIGGER_HEARTBEAT_MONITORING:
	 TYPE 				 = 
	 TIME_MAX_WAIT			 = -1
	 SLEEP_THREAD_TIME		 = -1
============================================
ID: 0 É servidor com clientes: 0 
Can not recover last checkpointing global
============================================
-----	 FAULT TOLERANCE 	-----
-----	 Saving the current settings 	-----
============================================
0
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 1		-----
============================================
TIME_TO_SAVE_GLOBAL = 45830[ns]
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 2		-----
============================================
TIME_TO_SAVE_GLOBAL = 180170[ns]
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 3		-----
============================================
TIME_TO_SAVE_GLOBAL = 180051[ns]
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 4		-----
============================================
TIME_TO_SAVE_GLOBAL = 216790[ns]
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 5		-----
============================================
TIME_TO_SAVE_GLOBAL = 163420[ns]
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 6		-----
============================================
TIME_TO_SAVE_GLOBAL = 208881[ns]
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 7		-----
============================================
TIME_TO_SAVE_GLOBAL = 250691[ns]
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 8		-----
============================================
TIME_TO_SAVE_GLOBAL = 194251[ns]
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 9		-----
============================================
TIME_TO_SAVE_GLOBAL = 193641[ns]
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 10		-----
============================================
TIME_TO_SAVE_GLOBAL = 199850[ns]
TEST FINALIZED OK
PASSOU AQUI FDP!
============================================
-----	 FAULT TOLERANCE 	-----
 FT_FOLDER 				 = ./data
 CHECKPOINTING_TIME_GLOBAL 		 = -1
 CHECKPOINTING_TIME_LOCAL 		 = -1
 CHECKPOINTING_TIME_DIFF_LOCAL 		 = -1
 CHECKPOINTING_GLOBAL_ITERATION 	 = 1
 TIME_SHARING_LOCAL_DATA_NEIGHBOR 	 = -1
 CHECKPOINTING_LOCAL_RECENT 		 = 1
 WORKLOAD_SCHEDULER 			  = DS
 WORKLOAD_SCHEDULER_LEADER 		 = -1
 TRIGGER_SIGNAL 			 = 0
 TRIGGER_HEARTBEAT_MONITORING:
	 TYPE 				 = 
	 TIME_MAX_WAIT			 = -1
	 SLEEP_THREAD_TIME		 = -1
============================================
ID: 0 É servidor com clientes: 0 
Can not recover last checkpointing global
============================================
-----	 FAULT TOLERANCE 	-----
-----	 Saving the current settings 	-----
============================================
0
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 1		-----
============================================
TIME_TO_SAVE_GLOBAL = 50841[ns]
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 2		-----
============================================
TIME_TO_SAVE_GLOBAL = 177500[ns]
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 3		-----
============================================
TIME_TO_SAVE_GLOBAL = 173031[ns]
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 4		-----
============================================
TIME_TO_SAVE_GLOBAL = 172150[ns]
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 5		-----
============================================
TIME_TO_SAVE_GLOBAL = 210351[ns]
PASSOU AQUI FDP!
============================================
-----	 FAULT TOLERANCE 	-----
 FT_FOLDER 				 = ./data
 CHECKPOINTING_TIME_GLOBAL 		 = -1
 CHECKPOINTING_TIME_LOCAL 		 = -1
 CHECKPOINTING_TIME_DIFF_LOCAL 		 = -1
 CHECKPOINTING_GLOBAL_ITERATION 	 = 1
 TIME_SHARING_LOCAL_DATA_NEIGHBOR 	 = -1
 CHECKPOINTING_LOCAL_RECENT 		 = 1
 WORKLOAD_SCHEDULER 			  = DS
 WORKLOAD_SCHEDULER_LEADER 		 = -1
 TRIGGER_SIGNAL 			 = 0
 TRIGGER_HEARTBEAT_MONITORING:
	 TYPE 				 = 
	 TIME_MAX_WAIT			 = -1
	 SLEEP_THREAD_TIME		 = -1
============================================
ID: 0 É servidor com clientes: 0 
Can recover last checkpointing global
GLOBAL DATA WAS SUCCESSFULLY READ IN 0
0ID_TIME_TO_READ_GLOBAL = 36340[ns]
DeLIA: Global Data read with sucess in 0
5
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 6		-----
============================================
TIME_TO_SAVE_GLOBAL = 4690795[ns]
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 7		-----
============================================
TIME_TO_SAVE_GLOBAL = 166300[ns]
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 8		-----
============================================
TIME_TO_SAVE_GLOBAL = 171511[ns]
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 9		-----
============================================
TIME_TO_SAVE_GLOBAL = 205591[ns]
============================================
-----	 DeLIA::SaveGlobalData 	-----
-----	 Iteration: 10		-----
============================================
TIME_TO_SAVE_GLOBAL = 217490[ns]
TEST FINALIZED OK
